### PR TITLE
To Do List - structure

### PR DESCRIPTION
**Pull request by Yordinia**
# To Do List - structure
In this pull request we have provided
- A todo list with adding and removing note functionalities
- Storing and retriving from local storage
image

### CODE REVIEW

File: script.js.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+// }
+// // Add event listeners to various elements on the page
+// form.addEventListener('submit', addItem);
+// todoList.addEventListener('click', toggleItem);
+// todoList.addEventListener('dblclick', editTaskDescription);
+// archiveBtn.addEventListener('click', clearCompleted);
+// refreshIcon.addEventListener('click', refreshPage);
+// downArrowIcon.addEventListener('click', addItem);
+// // Render the initial tasks on the page when it loads
+// renderTasks();
```
Comment (CollinsTatang) [inline script.js.js]: Hi @Yordinia,

- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

File: src/sccjj.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+// }
+// // Add event listeners to various elements on the page
+// form.addEventListener('submit', addItem);
+// todoList.addEventListener('click', toggleItem);
+// todoList.addEventListener('dblclick', editTaskDescription);
+// archiveBtn.addEventListener('click', clearCompleted);
+
+// enterIcon.addEventListener('click', addItem);
+// // Render the initial tasks on the page when it loads
+// displayTasks();
```
Comment (CollinsTatang) [inline src/sccjj.js]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

File: src/script.js [hunk 3]
```diff
@@ -1 +1,119 @@
+// Import the stylesheet
 import './style.css';
+
+// Select relevant HTML elements
+const form = document.getElementById('form'); // Get the form element
+const input = document.getElementById('new-item'); // Get the input element
+const todoList = document.getElementById('todo-list'); // Get the todo list element
+// const archiveBtn = document.getElementById('archive'); // Get the archive button element
```
Comment (CollinsTatang) [inline src/script.js:8]: - [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.

**Review by CollinsTatang — COMMENTED** [review]
Hi @Yordinia , 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional  README file.

- [ ] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.
- [ ] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [ ] Please remove the Microverse logo from your README file
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.
- [ ] Please your commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly correct that in your next commit massage.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Yordinia ,

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**